Makefile

```diff
@@ -52,3 +52,11 @@ publish: build ## Publish docker image
 clean: ## Clean autogenerated files/folders
 	@rm -rf bats
 	@rm -rf target
+
+create-release: ## Create github release given the APM Agent version if no tag release
+	@if [ -z "$(shell git tag -l v$(AGENT_VERSION))" ]; then \
+		echo "creating tag v$(AGENT_VERSION)"; \
+		gh release create "v$(AGENT_VERSION)" --title="$(AGENT_VERSION)" --generate-notes; \
```

Comment:
This make goal is triggered in GitHub runner - GitHub runners are configured with `gh` automatically, see https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#cli-tools

I decided to create a Make goal to help with testing it locally, otherwise, using the GitHub actions for that, it will be a pain in the heck.